src/supporting-data/taxonomy/config/TaxonomyColumnConfiguration.tsx

```diff
@@ -84,7 +84,7 @@ TaxonomyColumnConfiguration.set(TaxonomyColumn.lineage, {
   label: 'Lineage',
   // TODO: modify when we have a common approach to represent lineages
   render: ({ lineage }) =>
-    lineage
+    (lineage as Lineage)
       ?.filter(({ hidden }) => !hidden)
```

Comment:
By the way, I've noticed that when it's an object, the order should be inverted as the highest-level taxon will be last
```suggestion
      ?.filter(({ hidden }) => !hidden).reverse()
```